DQM/L1TMonitorClient/src/SealModule.cc

```diff
@@ -27,5 +27,11 @@ DEFINE_FWK_MODULE(L1EmulatorErrorFlagClient);
 #include <DQM/L1TMonitorClient/interface/L1TStage2CaloLayer2DEClient.h>
 DEFINE_FWK_MODULE(L1TStage2CaloLayer2DEClient);
 
+#include <DQM/L1TMonitorClient/interface/L1TStage2CaloLayer2DEClientSummary.h>
+DEFINE_FWK_MODULE(L1TStage2CaloLayer2DEClientSummary);
+
 #include <DQM/L1TMonitorClient/interface/L1TStage2RatioClient.h>
 DEFINE_FWK_MODULE(L1TStage2RatioClient);
+
```

Comment:
This seems to have sneaked in since the development was maybe done on a 10_1_X release.